File: sdks/dotnet/src/Models/Staff.cs
using Newtonsoft.Json;

namespace Puxbay.SDK.Models
{
    public class Staff
    {
        [JsonProperty("id")]
        public string Id { get; set; }

        [JsonProperty("username")]
        public string Username { get; set; }

        [JsonProperty("full_name")]
        public string FullName { get; set; }

        [JsonProperty("email")]
        public string Email { get; set; }

        [JsonProperty("role")]
        public string Role { get; set; }

        [JsonProperty("branch")]
        public string Branch { get; set; }

        [JsonProperty("branch_name")]
        public string BranchName { get; set; }
    }
}

File: sdks/php/src/Models/Staff.php
<?php

namespace Puxbay\Models;

class Staff
{
    public function __construct(
        public readonly string $id,
        public readonly string $username,
        public readonly string $fullName,
        public readonly string $email,
        public readonly string $role,
        public readonly ?string $branch = null,
        public readonly ?string $branchName = null,
    ) {
    }

    public static function fromArray(array $data): self
    {
        return new self(
            id: $data['id'],
            username: $data['username'],
            fullName: $data['full_name'],
            email: $data['email'],
            role: $data['role'],
            branch: $data['branch'] ?? null,
            branchName: $data['branch_name'] ?? null,
        );
    }

    public function toArray(): array
    {
        return [
            'id' => $this->id,
            'username' => $this->username,
            'full_name' => $this->fullName,
            'email' => $this->email,
            'role' => $this->role,
            'branch' => $this->branch,
        ];
    }
}
